packages/widget/src/components/address-input/styles.ts

```diff
@@ -0,0 +1,45 @@
+import { css } from 'lit';
+
+export const styles = css`
+  .addressInputContainer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 0.5rem;
+    min-height: 7.75rem;
+    height: 100%;
+  }
+
+  .inputAddressContainer {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    min-height: 7.75rem;
+    height: 100%;
+    gap: 0.5rem;
+  }
+
+  .inputAddress {
+    border-radius: 1.5rem;
+    border: 0.063rem solid var(--zinc-200);
+    width: 100%;
+    height: 4.25rem;
```

Comment:
i find padding more future-proof. if everything is setup properly, height will be that same as if you used fixed height and you don't have to worry about changing height if let's say font-size on input changes. i always opt for 'fluid' options, because fixed ones usually have some edge-cases that break them